lib/pushmi_pullyu/cli.rb

```diff
@@ -160,28 +157,31 @@ def rotate_logs
     PushmiPullyu.reset_logger = false
   end
 
-  def run_tick_loop
-    while PushmiPullyu.server_running?
-      # Preservation (TODO):
-      item = @queue.wait_next_item
-
-      # add additional information about the error context to errors that occur while processing this item.
-      Rollbar.scoped(noid: item) do
-        begin
-          # Download AIP from Fedora, bag and tar AIP directory and cleanup after block code
-          PushmiPullyu::AIP.create(item) do |aip_filename|
-            # Push tarred AIP to swift API
-            deposited_file = @storage.deposit_file(aip_filename, options[:swift][:container])
-            # Log successful preservation event to the log files
-            PushmiPullyu::Logging.log_preservation_event(deposited_file)
-          end
-        rescue => e
-          Rollbar.error(e)
-          # TODO: we could re-raise here and let the daemon die on any preservation error, or just log the issue and
-          # move on to the next item.
+  def run_preservation_cycle
+    item = queue.wait_next_item
+
+    # add additional information about the error context to errors that occur while processing this item.
+    Rollbar.scoped(noid: item) do
+      begin
+        # Download AIP from Fedora, bag and tar AIP directory and cleanup after block code
+        PushmiPullyu::AIP.create(item) do |aip_filename|
+          # Push tarred AIP to swift API
```

Comment:
It's basically extra context added to the alert for any exception that's thrown within the scope. So anything that blows up in the noid scope gets tagged with the noid, etc.